provider/datagen/src/registry.rs

```diff
@@ -82,6 +82,7 @@ macro_rules! registry {
         fn test_paths_correct() {
             $(
                 $(
+                    #[cfg($feature)]
```

Comment:
Seems like an extra line?

Reply:
This is needed to feature-gate the marker usage.

Reply:
I see, it's an auto-generated test that previously failed without `--all-features` and now always fails since the dependency doesn't exist.